frontend/.gitignore

```diff
@@ -22,29 +22,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-.vscode# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
There were some duplicate lines in the frontend .gitignore (probably just copy-past issue) so I'm just removing them here.

Reply:
Wow, my bad, thank you for fixing that!

Could you add the bottom 2 lines back in though?

```
# CSS files output
*.output.css
```

Reply:
Oh whoops! Yeah, let me add those back in.

Reply:
✅